pkg/std/fs.go

```diff
@@ -70,7 +70,7 @@ func directoryListing(base, rel string) (Directory, error) {
 		return infos[i].Name() < infos[j].Name()
 	})
 
-	var files []FileInfo
+	files := make([]FileInfo, 0, len(infos))
```

Comment:
Length `0` because it must start genuinely empty (we won't add everything that's in `infos`), capacity `> 0` because it is most likely there will be some entries.